interfaces/builtin/unity8_calendar_test.go

```diff
@@ -176,20 +177,21 @@ func (s *Unity8CalendarInterfaceSuite) TestPermanentSlotSnippetAppArmor(c *C) {
 	c.Check(string(snippet), testutil.Contains, "name=\"org.gnome.evolution.dataserver.Sources5\"")
 }
 
+const unity8mockSlotInfoYaml = `name: cal
+version: 1.0
+slots:
+ cal:
+  interface: unity8-calendar
+apps:
+ app:
+  command: foo
+  slots:
+   - cal
```

Comment:
(nitpick²) Slightly inconsistent with most of the other tests where the interface name is used. Might make sense here too (i.e. "cal" -> "unity8-calendar").